src/utils/redis.ts

```diff
@@ -257,3 +257,9 @@ export const mget = ({
   keys: string[];
   client: Redis;
 }) => client.mget(...mgetKeys);
+
+export const get = ({ key, client }: { key: string; client: Redis }) =>
+  client.get(key);
+
+export const del = ({ key, client }: { key: string; client: Redis }) =>
+  client.del(key);
```

Comment:
why such wrapper? why just not do `client.del` in callee?

Reply:
Good question :) I initially started implementing these wrappers as a means to support both the [`node_redis`](https://github.com/NodeRedis/node-redis) and [`ioredis`](https://github.com/luin/ioredis) Redis clients as there were subtle and slight differences between the two which these wrappers abstracted. However, there are now a couple of places in the codebase where I am using features only supported by `ioredis` (Redis Cluster and `.defineCommand` for lua scripts) so these wrappers are mostly irrelevant now as Conveyor MQ will not be supporting `node_redis` for the time being.
I'll be stripping out some of these unnecessary wrappers in another PR.

Reply:
@glensc If you have any other thoughts, suggestions or questions on the rest of this Rate limiting PR or the rest of the code base, please do share :) It would be great to have your input.

Reply:
@jasrusable sorry, no, I don't. I don't even use the library :)

Reply:
If I'd need to solve a similar problem I'd probably use adapter pattern with decorator/delegation:
- https://en.wikipedia.org/wiki/Adapter_pattern#Usage

and I'd create class, not export single functions. but I've come from a different world than nodejs/typescript used here.